examples/helloworld/greeter_client/main.go

```diff
@@ -16,6 +16,7 @@
  *
  */
 
+// greeter_client command ...
```

Comment:
As this is a package level comment, it should start with "Package main ..."
```suggestion
// Package main implements a client for Greeter service.
```